### DISCUSSION TITLE
Welcome to Neo Launcher Discussions!

### DISCUSSION THREAD

**Discussion by saulhdev** [otakuhqz/Omega#118] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by earlwallace** [comment]
Good morning!  Installed launcher in android moto g stylus and all is working well.   I am trying to find the config for biometric app protection but not as of yet.  I saw in faq that it is in hidden app feature but can not find it there.  Perhaps a more visible config choice would be great.  I have used a lot of launchers over the years but never seen such a wonderful configuration settings menu.  Earl Wallace,  jjwwhhjr@hotmail.com

**Reply by saulhdev** [reply]
Enable protected app on settings > Drawer.  Then go to Settings > Drawer > Protected App Shortcuts.
Click on the security icon next to each app for one or more apps you want to protect.
